classes/Product.php

```diff
@@ -6260,6 +6269,13 @@ public function getWsManufacturerName()
         return Manufacturer::getNameById((int) $this->id_manufacturer);
     }
 
+	public function getWsSpecificPrices()
+	{
+		return Db::getInstance()->executeS('SELECT `id_specific_price` id, `id_product_attribute`
```

Comment:
The method name says it returns specifc prices, when the sql request return and `id` and an `id_product_attribute` :thinking:

Reply:
This is why a phpdoc block is required :smile:

Reply:
I can change the method name to getWsSpecificPricesIds() if you prefer... i named it like this to be consistent with the other webservice getter methods.

Reply:
If you can yes. The old way to defined method is not obvious, and we can't trust methods name :/

Reply:
Updated